src/CoreWCF.Primitives/src/CoreWCF/Description/DispatcherBuilder.cs

```diff
@@ -505,8 +505,6 @@ private static Type[] GetSupportedChannelTypes(ContractDescription contractDescr
                 }
             }
 
-            BindOperations(contractDescription, null, dispatch);
```

Comment:
what was the reason for call and why did you remove ?

Reply:
This call causes the behavior OperationInvokerBehavior to be applied which sets DispatchOperation.Invoker. This method was being called twice which normally isn't a problem, unless you try to replace the Invoker. In which case applying OperationInvokerBehavior a second time causes it to repalce the Invoker with the default implementation again. It was a bug applying this twice.